scripts/generate-javadoc-plugins.groovy

```diff
@@ -24,6 +24,20 @@ def jsonUrlMap = [:]
 // define sort order for plugins
 def keyComparator = [compare: { e1, e2 -> e1.key.compareToIgnoreCase(e2.key) }] as Comparator
 
+Set<String> pluginsAIDs = null;
+if (args.length > 0 && !args[0].trim().empty) {
+    println "Args: ${args.length}, values=${args}"
+    if (pluginsAIDs == null) {
+        pluginsAIDs = new HashSet<>(args.length)
+    }
+
+    def argPluginIDs = args[0].split()
+    for (def pluginID : argPluginIDs) {
+        pluginsAIDs.add(pluginID)
```

Comment:
I think it would be good if `subversion, git` worked. AFAICT the leading space for ` git` is not being trimmed.